diff --git a/node_modules/expo-image/ios/ContentPosition.swift b/node_modules/expo-image/ios/ContentPosition.swift
index 73c34ca..905f068 100644
--- a/node_modules/expo-image/ios/ContentPosition.swift
+++ b/node_modules/expo-image/ios/ContentPosition.swift
@@ -10,7 +10,7 @@ typealias ContentPositionValue = Either<Double, String>
  - Note: Its intention is to behave like the CSS [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) property.
  */
 struct ContentPosition: Record {
-  static let center = Self()
+  nonisolated(unsafe) static let center = Self()
 
   @Field
   var top: ContentPositionValue?
diff --git a/node_modules/expo-image/ios/ImageLoadOptions.swift b/node_modules/expo-image/ios/ImageLoadOptions.swift
index 50099ae..1e8061a 100644
--- a/node_modules/expo-image/ios/ImageLoadOptions.swift
+++ b/node_modules/expo-image/ios/ImageLoadOptions.swift
@@ -2,7 +2,7 @@
 
 import ExpoModulesCore
 
-internal struct ImageLoadOptions: Record {
+internal struct ImageLoadOptions: Record, @unchecked Sendable {
   @Field var maxWidth: Int?
   @Field var maxHeight: Int?
   @Field var tintColor: UIColor? = nil
diff --git a/node_modules/expo-image/ios/ImageSource.swift b/node_modules/expo-image/ios/ImageSource.swift
index 4d23e20..9b069c7 100644
--- a/node_modules/expo-image/ios/ImageSource.swift
+++ b/node_modules/expo-image/ios/ImageSource.swift
@@ -2,7 +2,7 @@
 
 import ExpoModulesCore
 
-struct ImageSource: Record {
+struct ImageSource: Record, @unchecked Sendable {
   @Field
   var width: Double = 0.0
 
